src/analyses/apron/apronAnalysis.apron.ml

```diff
@@ -125,6 +125,7 @@ struct
       )
       else (
         let v_out = Goblintutil.create_var @@ makeVarinfo false (v.vname ^ "#out") v.vtype in (* temporary local g#out for global g *)
+        v_out.vattr <- v.vattr; (*copy the attributes*)
```

Comment:
Clearly this is copying the attributes, but the comment should explain why this is now needed.